packages/table/src/regions.ts

```diff
@@ -467,6 +467,16 @@ export class Regions {
         return true;
     }
 
+    public static isRegionValidForTable(region: IRegion, numRows: number, numCols: number) {
+        if (region.rows != null && (region.rows[0] >= numRows || region.rows[1] >= numRows)) {
```

Comment:
It seemed a prudent check to make, given that we're checking that it's in-bounds. You're right that it's probably not necessary, though -- you can have negative regions. I can remove it if you think it's a bad thing to have/is confusing.